extensions/markdown-language-features/media/markdown.css

```diff
@@ -105,6 +105,18 @@ body.showEditorSelection li.code-line:hover:before {
 .vscode-high-contrast.showEditorSelection .code-line .code-line:hover:before {
 	border-left: none;
 }
+ul ul{
+	margin-bottom: 0%;
+}
+ul ol{
```

Comment:
[nit] Just use `0` instead of `0%`. They mean the same thing but zero without units is much more common